Vizzuality/heco-invest
# feat: Adding impact filter to project maps endpoint

I have double checked projects API and projects map API so they use same service for filtering/searching and allow same params. Project map endpoint missed `impact` filter so I allowed it there. There is no extra test, because both endpoints use `API::Filterer.new(projects, filter_params.to_h).call` which is tested quite well ;)

## Testing instructions

Just check that same params can be send to project and map endpoint at rswag doc.

## Tracking

https://vizzuality.atlassian.net/browse/LET-1034
